src/bpftrace.cpp

```diff
@@ -1615,9 +1615,9 @@ std::string BPFtrace::resolve_usym(uintptr_t addr, int pid, bool show_offset, bo
   struct bcc_symbol_option symopts;
```

Comment:
Can we `memset` this to 0 to protect against future additions receiving junk values? The designated initializers zero out everything unset IIRC.